packages/sdk-bridge/src/backend.ts

```diff
@@ -0,0 +1,281 @@
+import {  GoldSkyBackend, GoldSkyMessage, MessageFilter } from "@nomad-xyz/sdk";
+import {MessageBackend} from "@nomad-xyz/sdk";
```

Comment:
Can add this import to the above, looks like we need to lint as well.

Reply:
Addressed in the new commit